tatanka/client/mesh/mesh.go

```diff
@@ -152,6 +152,8 @@ func (m *Mesh) emit(thing any) {
 
 func (m *Mesh) handleTatankaRequest(route string, payload json.RawMessage, respond func(any, *msgjson.Error)) {
 	switch route {
+	case mj.RouteNegotiate:
+		m.handleNegotiate(payload, respond)
```

Comment:
This should be a peer request instead of a tatanka request. Also should this be `ProposeMatch`?

Reply:
Yeah I'm probably not using the comms right at all, is there an example somewhere of a peer request already?

Reply:
Kind of, in `conn_live_test.go`, but you just have to add a route in `handlePeerRequest` rather than `handleTatankaRequest`.

Reply:
Does this look more correct? https://github.com/decred/dcrdex/compare/f6de84ceef070bd0c4889eca78dba7beadf4ae8d..6f55b24e340f0987fe2b562f70431eca13241fc3

Reply:
Yeah that looks good.